appveyor.yml

```diff
@@ -27,5 +27,4 @@ install:
 build: false
 
 test_script:
-  - "%CMD_IN_ENV% conda build --quiet devtools\\conda-recipe"
```

Comment:
If this is unnecessary, you can remove the entire appveyor-ci folder in devtools and the definitions earlier this file under "global"